Signal/src/call/CallService.swift

```diff
@@ -75,6 +75,7 @@ enum CallError: Error {
     case disconnected
     case externalError(underlyingError: Error)
     case timeout(description: String)
+    case unexpected(description: String)
```

Comment:
What's your intention for how `unexpected` should be used in distinction to the existing `assertionError`? They seem really close.